db-migrations/23_version_labels.down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE versions DROP COLUMN labels;
+
+DROP INDEX version_labels_idx;
```

Comment:
I think we need to change this order. Drop index should be performed before dropping the column because dropping the labels column will usually delete the related index, so the drop index command will fail.

Reply:
You can dry run the schema rollback locally like so:
> Using migrate command from https://github.com/golang-migrate/migrate
```
migrate -source file://./db-migrations -database postgres://USER:PASSWORD@localhost:5432/DATABASE_NAME?sslmode=disable down 1
```

Reply:
will edit this.